File: leetcode-submissions/268_Missing_Number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n=nums.size();
        int expectedsum=0;
        for(int i=1;i<=n;i++){
            expectedsum+=i;
        }
        int actualsum=0;
        for(int ele:nums){
            actualsum+=ele;
        }
        int missingnum=expectedsum-actualsum;
        return missingnum;
    }
};

File: leetcode-submissions/268_Missing_Number.ts
function missingNumber(nums: number[]): number {
   let actualsum=0;
   let expectedsum=0;
   for(let i=0;i<nums.length;i++){
    actualsum+=nums[i];

   }
   for(let i=1;i<=nums.length;i++){
    expectedsum+=i;
   } 
   return expectedsum-actualsum;
};